openedx/core/djangoapps/content/course_structures/migrations/0003_truncate_course_structures.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        # Clear all of the course structure data so that it can be regenerated with the updated CompressedTextField.
+        db.clear_table('course_structures_coursestructure')
+
+    def backwards(self, orm):
+        # There is no going back!
+        pass
```

Comment:
How expensive is it to regenerate all CourseStructure data using the generate_course_structure management command? I could see this backwards-incompatible migration throwing off a release master, especially with a comment like "There is no going back!".